internal/cache/helper_test.go

```diff
@@ -4,13 +4,14 @@ type userInfos struct {
 	name     string
 	uid      int64
 	password string
+	gecos    string
```

Comment:
Thanks for adding the gecos! (However, this could have been a separated PR to really see the golden noise diff more easily)

Reply:
Yeah... I didn't like the amount of noise diff that was generated, but there was a golden file that was empty because the users_in_db didn't have any gecos value, so I thought we might as well just change everything and deal with the noise anyway, specially since I'm switching the db that is used is some tests.